android/plugins/fresco/build.gradle

```diff
@@ -21,7 +21,6 @@ android {
         implementation project(':android')
         implementation deps.fresco
         implementation deps.frescoFlipper
-        implementation deps.frescoStetho
```

Comment:
This doesn't seem to be quite covered: https://travis-ci.org/facebook/flipper/jobs/642213519?utm_medium=notification&utm_source=github_status

I wonder if this is just a transitive dependency and not actually a dependency on the Stetho parts in it: https://repo1.maven.org/maven2/com/facebook/fresco/stetho/2.0.0/stetho-2.0.0.pom

Reply:
Had a chat with @oprisnik from Fresco. Currently, the Cache class causing the failure is part of the Stetho artefact, but for no other reason than convenience when we first did this. He'll try to find somebody to create a separate package for this.

In the meantime, @ZacSweers, would you mind trying to use an exclude rule for on this dependency on this? I haven't tried it but excluding the Stetho dep should still work, e.g.

```groovy
implementation(deps.frescoStetho) {
   exclude group: 'com.facebook.stetho'
}
```

(From memory, might be the wrong syntax.)

Reply:
Updated 👍  d4390d3

Reply:
Thanks. That should work for now. We'll do a refactor on our side soon so that our Flipper plugin doesn't depend on Stetho. Once we've released a new Fresco version with that fix, this can be simplified again :)

Reply:
The Stetho dep has been removed: https://github.com/facebook/fresco/commit/d1d5bdaf8c37058bf965920d7caa461a8402ef98

The next Fresco release will include this change.